test/bun.js/buffer.test.js

```diff
@@ -600,6 +600,84 @@ it("Buffer.from(base64)", () => {
   ).toBe('console.log("hello world")\n');
 });
 
+it("Buffer.swap16", () => {
+  const examples = [
+    ["", ""],
+    ["a1", "1a"],
+    ["a1b2", "1a2b"]
+  ];
```

Comment:
Can we also copy node's tests for this? https://github.com/nodejs/node/blob/master/test/parallel/test-buffer-swap.js#L1-L153